src/aeso_code_errors.erl

```diff
@@ -41,39 +41,17 @@ format({invalid_entrypoint, Why, Ann, {id, _, Name}, Thing}) ->
     Msg = io_lib:format("The ~sof entrypoint '~s' ~s.",
                         [ThingS, Name, Bad]),
     case Why of
-        polymorphic  -> mk_err(pos(Ann), Msg, "Use the FATE backend if you want polymorphic entrypoints.\n");
         higher_order -> mk_err(pos(Ann), Msg)
     end;
-format({cant_compare_type_aevm, Ann, Op, Type}) ->
-    StringAndTuple = [ "\n- type string"
-                       "\n- tuple or record of word type" || lists:member(Op, ['==', '!=']) ],
-    Msg = io_lib:format("Cannot compare values of type\n"
-                        "~s\n"
-                        "The AEVM only supports '~s' on values of\n"
-                        "- word type (int, bool, bits, address, oracle(_, _), etc)"
-                        "~s",
-                        [pp_type(2, Type), Op, StringAndTuple]),
-    Cxt = "Use FATE if you need to compare arbitrary types.",
-    mk_err(pos(Ann), Msg, Cxt);
 format({invalid_aens_resolve_type, Ann, T}) ->
     Msg = io_lib:format("Invalid return type of AENS.resolve:\n"
                         "~s\n"
                         "It must be a string or a pubkey type (address, oracle, etc).",
                         [pp_type(2, T)]),
     mk_err(pos(Ann), Msg);
-format({unapplied_contract_call, Contract}) ->
-    Msg = io_lib:format("The AEVM does not support unapplied contract call to\n"
-                        "~s", [pp_expr(2, Contract)]),
-    Cxt = "Use FATE if you need this.",
-    mk_err(pos(Contract), Msg, Cxt);
-format({unapplied_builtin, Id}) ->
-    Msg = io_lib:format("The AEVM does not support unapplied use of ~s.", [pp_expr(0, Id)]),
-    Cxt = "Use FATE if you need this.",
-    mk_err(pos(Id), Msg, Cxt);
 format({invalid_map_key_type, Why, Ann, Type}) ->
```

Comment:
I don't see this error being thrown anywhere in the code. And I would say the same for the error `last_declaration_must_be_main_contract `, it's also never used anywhere in the code.